wolfcrypt/src/ecc.c

```diff
@@ -7328,10 +7328,12 @@ static int ecc_check_pubkey_order(ecc_key* key, ecc_point* pubkey, mp_int* a,
         if (err == MP_OKAY && !wc_ecc_point_is_at_infinity(inf))
             err = ECC_INF_E;
 #else
+	{
```

Comment:
Spacing wrong here - don't use tabs.

Reply:
Oh dear.  Fixed.  Ironically, I had already taken the initiative to fix a bunch of broken whitespace in api.c in this very PR...

I have `wsErrorHighlight = all` in my ~/.gitconfig but it still won't spotlight tabs in files with non-tab indentation.  For me, stray tabs are entirely a function of forgetting to `M-x wolfssl-mode-on` the buffer in emacs.  And there's actually quite a few in the old codebase, but I fix them when I find them.